tests/data_processing/test_spatialaverage.py

```diff
@@ -9,12 +9,8 @@
 
 def test_spatialaverage() -> None:
     original_dir = Path.cwd()
```

Comment:
I would use this to resolve the path directory containing the script: `pathlib.Path(__file__).resolve().parent`

Reply:
think i don't get this one